tests/devapps/MSI/HelperService/MSIHelperService/ManagedIdentityResponse.cs

```diff
@@ -6,27 +6,45 @@
 
 namespace MSIHelperService
 {
+    /// <summary>
+    /// ManagedIdentityResponse
+    /// </summary>
     [JsonObject]
     public class ManagedIdentityResponse
```

Comment:
I think Neha added some error fields to this one recently.

Reply:
There is a ManagedIdentityErrorResponse for error fields

Reply:
The only reason we use `ManagedIdentityResponse` is to trim the access token, else we can send the response as is so MSAL can handle it. But for error response there is nothing to filter so we do not transform and send it to MSAL as is